### PR TITLE
Updated Kamael To latest Version

### PR DESCRIPTION
**Pull request by oxrock**
Bunch of little improvements, this version is largely the same as the one seen during the rlbot championship 2020 tournament. The biggest difference is that Kamael now transitions from ground dribble into air dribbles much more frequently, especially in 1's.

### CODE REVIEW

File: RLBotPack/Kamael/Peter.cfg [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 looks_config = ./peter_appaearance.cfg
 maximum_tick_rate_preference = 120
-#supports_early_start = True
+supports_early_start = False
 # Path to python file. Can use relative path from here.
 python_file = ./Kamael.py
 
 # Name of the bot in-game
-name = St. Peter
+name = St. Peter_dev
```
Comment (tarehart) [inline RLBotPack/Kamael/Peter.cfg:10]: Unintentional name change I suspect

**Comment by tarehart** [comment]
Transitioning from ground dribble into air dribbles sounds awesome

File: RLBotPack/Kamael/Peter.cfg (hunk 1 above)
Comment (oxrock) [reply RLBotPack/Kamael/Peter.cfg:10]: oh, nice catch. Yeah. Simply forgot I'd done that.

Comment (oxrock) [reply RLBotPack/Kamael/Peter.cfg]: It should be fixed now.

**Review by tarehart — APPROVED** [review]
Thanks for updating!

**Pull request merged**